net/http_client.h

```diff
@@ -11,11 +11,8 @@
 #define NOMINMAX
 #include <winsock2.h>
 #else
-#if defined(__FreeBSD__) || defined(__SYMBIAN32__)
```

Comment:
Symbian is often a bit weird.  I think it's probably safest to leave that part in unless you've tested the Symbian build.

-[Unknown]